deployment/test/services.jrl

```diff
@@ -23,3 +23,47 @@ p({
     ]
   }
 })
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"appConfig",
+  "lazy":false,
+  "parameters":false,
+  "service": {
+    "class":"foam.nanos.app.AppConfig",
+    "mode":4
+  }
+})
+p({
+  class: "foam.nanos.boot.NSpec",
+  name: "smtpAgent",
+  lazy: false,
+  service: {
+    class: "foam.nanos.notification.email.test.MockSMTPAgent",
+    id: "test"
+  }
+})
+
+p({
+  class: "foam.nanos.boot.NSpec",
+  name: "dugLoopbackDAO",
```

Comment:
Can we serve the dao?

```suggestion
  name: "dugLoopbackDAO",
  serve: true,
```

will also needs
```
"client":
  """
    {
      "of":"foam.nanos.dig.DUGLoopback"
    }
  """
```

Reply:
Why?

Reply:
I was testing locally and it's handy to see the loopback records in the table view.